website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,190 @@
+[DEBUG] generating authentication headers
+[DEBUG] creating request
+[DEBUG] sending request
+[DEBUG] performing request: method=POST url=https://census.license.hashicorp.services
+[DEBUG] recording audit record
+[INFO]  {"payload":{"payload_version":"1","license_id":"974b62bd-7e7b-4556-7741-9683ecd9acb4","product":"vault","product_version":"1.2.3","export_timestamp":"2023-04-14T15:06:35.258709-04:00","snapshots":[{"snapshot_version":1,"snapshot_id":"0001J3K91VB1SYJGSHM9JTFXJ8","process_id":"01GY0JJYXZ6MY01GNM9VN88ZEJ","timestamp":"2023-04-14T15:06:35.257143-04:00","schema_version":"0.1.0","service":"vault","metrics":{"clientcount.current_month_estimate":{"key":"clientcount.current_month_estimate","kind":"sum","mode":"write","value":117,"labels":{"type":{"entity":58,"nonentity":59}}},"clientcount.previous_month_complete":{"key":"clientcount.previous_month_complete","kind":"sum","mode":"write","value":300,"labels":{"type":{"entity":100,"nonentity":200}}}}}],"metadata":{"vault":{"billing_start":"2023-03-24T00:00:00Z"}}}}
+[DEBUG] completed recording audit record
+[DEBUG] export finished successfully"
+```
+
+</CodeBlockConfig>
+
+If your installation is air-gapped or your network doesn’t allow the correct
+egress, logs will show an error. 
+
+<CodeBlockConfig hideClipboard>
+
+```
+[DEBUG] reporting: beginning snapshot export
+[DEBUG] reporting: creating payload
+[DEBUG] reporting: marshalling payload to json
+[DEBUG] reporting: generating authentication headers
+[DEBUG] reporting: creating request
+[DEBUG] reporting: sending request
+[DEBUG] reporting: performing request: method=POST url=https://census.license.hashicorp.services
+[DEBUG] reporting: error status code received: statusCode=403
+```
+
+</CodeBlockConfig>
+
+Reconfigure your network to allow egress and check back in 24 hours. 
+
+## Opt out
+
+If your installation is air-gapped, or you want to manually collect and report
+on the same entitlement utilization metrics, you can opt-out of automated
+reporting. 
+
+Manually reporting these metrics can be time consuming and does not prevent you
+from sharing the minimum data required to validate entitlement utilization. If
+you are considering opting out because you’re worried about the data, we
+strongly recommend that you review the [example payloads](#example-payloads)
+before opting out. If you have concerns with any of the automatically-reported
+data please bring them to your account representative. 
+
+You have two options to opt out of automated reporting: HCL configuration
+(recommended) Environment variable (requires restart)
+
+Opting out in your product’s configuration file doesn’t require a system
+restart, and is the method we recommend. Add the following block to your server
+configuration file (e.g. `vault-config.hcl`).
+
+```hcl
+reporting {
+	license {
+		enabled = false
+   }
+}
+```
+
+If you need to, you can also opt out using an environment variable. This option
+requires a system restart. 
```

Comment:
Nit: "system restart" makes me think of restarting the server itself (rather than the Vault process). Should we maybe say "Vault restart" or "process restart" isntead?